CHANGELOG.md

```diff
@@ -4,6 +4,19 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [20.8.4] (unreleased)
+### Added
+
+### Changed
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
+[Unreleased]: https://github.com/greenbone/gsa/compare/v20.8.3...v20.8.2
```

Comment:
```suggestion
[Unreleased]: https://github.com/greenbone/gsa/compare/v20.8.4...v21.4.0
```

Reply:
This produces a broken link, doesn't it? In the past we had at least a working link here and fixed it when releasing

Reply:
And why should be compare it to 21.4.0?

Reply:
ahh I did it wrong ... wait ...

Reply:
nvm ... now it should be correct ... I guess?